drivers/clock_control/nrf_clock_calibration.c

```diff
@@ -7,7 +7,7 @@
 #include <drivers/clock_control.h>
 #include "nrf_clock_calibration.h"
 #include <drivers/clock_control/nrf_clock_control.h>
-#include <hal/nrf_clock.h>
+#include <nrfx_clock.h>
```

Comment:
Without `#include <nrfx_power.h>` samples for 5340 will not compile as `nrfx_power.h` includes `"nrfx_power_compat.h"` with usbreg stuff (for 5340).